### ISSUE TITLE
Cannot read property 'memoryOnly' of undefined

### ISSUE THREAD

**Issue by bkuri** [nuxt-community/firebase-module#224]
### Version
@nuxtjs/firebase: 6.0.0
firebase: 7.15.0
nuxt: 2.12.2

### Steps to reproduce
Upgrade `@nuxtjs/firebase ^5.2.0 → ^6.0.0`
Upgrade `firebase ^7.14.5 → ^7.15.0`

### What is Expected?
`nuxt build` to work

### What is actually happening?
```
yarn run v1.22.4
$ nuxt

   ╭──────────────────────────────────────────╮
   │                                          │
   │   Nuxt.js v2.12.2                        │
   │   Running in development mode (spa)      │
   │                                          │
   │   Listening on: http://localhost:3000/   │
   │                                          │
   ╰──────────────────────────────────────────╯

ℹ Preparing project for development                                                 22:01:08
ℹ Initial build may take a while                                                    22:01:08
✔ Builder initialized                                                               22:01:08

 ERROR  Could not compile template ...node_modules/@nuxtjs/firebase/lib/plugins/main.js: Cannot read property 'memoryOnly' of undefined

  at Promise.all.templateFiles.map (node_modules/@nuxt/builder/dist/builder.js:6018:17)
```

### What fixes it?
Downgrading `@nuxtjs/firebase` to `5.2.0`

**Comment by bkuri** [comment]
Seems to be related to #135

**Comment by lupas** [comment]
Thanks! Fixed with v6.0.1.
